index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./css/styles.css">
+    <script src="https://kit.fontawesome.com/cebf050324.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <div class="background">
+      <nav>
+        <ul class="list-format font-color">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+      <header>
+          <h1>Howdy! I am Flover. <span> I am a software developer</span></h1>
+          <p class="font-color">
+             I can assist you building the product, feature, or website you may look
+             for, I have a passion for software development, please review some of
+             my works and experience! if you like what you see and have a project
+             you need coded do not hesitate to contact me.
+          </p>
+          <ul class="list-format font-color">
+            <li><a href="https://github.com/flov3rh" class="fa-brands fa-square-github"></a></li>
+            <li><a href="https://www.linkedin.com/in/flov3rh/" class="fa-brands fa-linkedin"></a></li>
+            <li><a href="https://twitter.com/flov3rh" class="fa-brands fa-twitter"></a></li>
+            <li><a href="#" class="fa-solid fa-envelope"></a></li>
+          </ul>
```

Comment:
- hey, the social media icon looks different from that on Figma. please, kindly use the same icon as used in the Figma design to make your design consistent following the given template.


![image](https://github.com/flov3rh/portfolio/assets/59397159/3f7079c2-151b-485c-8465-989a87bad26f)


![image](https://github.com/flov3rh/portfolio/assets/59397159/35a8558b-bd38-4f91-964e-45980b087fcd)

Reply:
![image](https://github.com/flov3rh/portfolio/assets/42297973/2544000b-3947-48aa-904f-3cbe7b19e1cb)

the template shows me a different order, all icons are linked to my accounts however I do not have an account with the peace sign logo hence why was ommited